ElectronClient/InteropServiceHelper.js

```diff
@@ -92,6 +95,17 @@ class InteropServiceHelper {
 		return this.exportNoteTo_('printer', noteId, options);
 	}
 
+	static async defaultFilename(noteIds, fileExtension) {
+		const note = await Note.load(noteIds[0]);
+		const folder = await Folder.load(note.parent_id);
```

Comment:
It would be best to check that note and folder are not null. Note probably isn't, but folder could be in rare cases. If null I guess you could use the ID as default.